packages/functions/src/coordinator/GetCoordinatorHandler.ts

```diff
@@ -11,11 +12,15 @@ export default async function (
     console.error("Could not find calling user", callerId);
     throw Error(`User is not a coordinator`);
   }
+  const coordinatorDonorUser = await DonorDAL.getDonor(callerId);
 
   return {
     coordinator: {
       role: coordinator.role,
       activeHospitalsForCoordinator: getCoordinatorHospitals(coordinator),
+      name: coordinatorDonorUser
```

Comment:
Can this really be undefined?
Do we support coordinators who's donors have no names?

Reply:
Safety?

Reply:
I don't think this is needed, your call